### PR TITLE
Added Title and text card issue #15

### PR DESCRIPTION
**Pull request by Kiran-Serolla**
# Description

This PR is to create a storybook card  with Title and Text .

Fixes # 15

# How to test?
it is tested in storybook and is working **http://localhost:6007/?path=/story/title-and-text-card--component**

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Review by Pallabi-ghosh — DISMISSED** [review]

File: src/client/components/Card/Card.styles.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+.title-text-card {
+  box-sizing: border-box;
+  position: absolute;
```
Comment (tpetrina) [inline src/client/components/Card/Card.styles.css:3]: Are you sure you need `absolute` positioning? Seeps a bit off.

File: src/client/components/Card/Card.styles.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  border: 1px solid #d5d5d5;
+  border-radius: 15px;
+  background-color: #f6f6f6;
+  color: #666666;
+  overflow: auto;
+  font-family: Roboto, sans-serif;
+  width: 282px;
```
Comment (tpetrina) [inline src/client/components/Card/Card.styles.css:10]: Not sure that we want to specify exact width. Are you sure? Have you tested on all screen sizes? Remove and we will later add it if needed.

**Review by tpetrina — CHANGES_REQUESTED** [review]
Just touch CSS a bit and you are good to go!

**Review by Pallabi-ghosh — APPROVED** [review]

**Review by tpetrina — APPROVED** [review]
Good work!

**Pull request merged**